### ISSUE TITLE
Hidden element identified as “About blank hash link” + “Share this publication” button visible focus 

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#621]
When viewing the summary page for a publication, there is a “Share this publication” button at the bottom of the view, but this button does not receive visible focus when navigating to it using Talkback. Also, there is another element being identified just before the share button that Talkback identifies as “About blank hash link”, which also does not receive visible focus and has the same function as the “Share this publication” button.

Remove the hidden element identified as “About blank hash link” and ensure that the “Share this publication” button receives visible focus when navigating to it using Talkback.

![GitHub34](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/4d3a4dae-eedb-4011-9d20-2e5b61b3385e)
